bundles/net.tourbook/src/net/tourbook/messages.properties

```diff
@@ -868,10 +868,10 @@ Dlg_TourMarker_MsgBox_delete_markers_message     = Delete markers ?\n\
                                                    {0}
 Dlg_TourMarker_MsgBox_delete_markers_title       = Delete Markers
 
-Enum_Cadence_None = none
-Enum_Cadence_Rpm = rpm
-Enum_Cadence_Spm = spm
-Enum_Cadence_Invalid = invalid
+App_Cadence_None = none
+App_Cadence_Rpm = rpm
+App_Cadence_Spm = spm
+App_Cadence_Invalid = invalid
```

Comment:
When you use the messages editor http://mytourbook.sourceforge.net/mytourbook/index.php/development/translation/install-resource-editor then the text strings are automatically sorted and formatted. This issue is not a problem because when I use this editor for this file, it is formatted. When you use the messages editor then you have to configure the formatting as described in the docu otherwise a code review has 100% differences